framework/test/unit/modules/interoperability/mainchain/commands/terminate_sidechain_for_liveness.spec.ts

```diff
@@ -100,6 +100,18 @@ describe('TerminateSidechainForLivenessCommand', () => {
 			jest.spyOn(interopMod['internalMethod'], 'isLive').mockResolvedValue(true);
 		});
 
+		it('should return error validation fails', async () => {
```

Comment:
we are explicitly adding tests for command schema using `validator.validate` since the `validator.validate` is not called in the `command.verify` anymore